src/app/clusters/test-cluster-server/test-cluster-server.cpp

```diff
@@ -237,6 +335,11 @@ CHIP_ERROR TestAttrAccess::ReadListNullablesAndOptionalsStructAttribute(Attribut
     });
 }
 
+CHIP_ERROR TestAttrAccess::WrtieListNullablesAndOptionalsStructAttribute(TLV::TLVReader & aReader)
+{
+    // TODO Add yaml test case for NullablesAndOptionalsStruct list
```

Comment:
Anything blocking us from doing this in this PR? I'd rather not file issues if we can avoid it (that then disappear...)

Reply:
We don't have yaml test case and implementation for read now, also some codegen issue I can't resolve, need to address them all together